QUICKSTART/QUICKSTART.md

```diff
@@ -0,0 +1,258 @@
+
+# Quickstart for VSCODE-CODE-COVERAGE
```

Comment:
We should link this document from the readme that is exposed on the vscode marketplace. (https://github.com/ryanluker/vscode-coverage-gutters/blob/master/README.md)